### PR TITLE
Fix: Replace missing modified_by with placeholder user

### PR DESCRIPTION
**Pull request by JustinHoldstock**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @JustinHoldstock has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #519

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/api/Versions.js [hunk 1]
```diff
@@ -37,10 +38,14 @@ class Versions extends OffsetBasedAPI {
...
         const { entries } = data;
-        const versions = entries.map((version: BoxItemVersion) => ({
-            ...version,
-            action: version.trashed_at ? ACTION.delete : ACTION.upload
-        }));
+        const versions = entries.map((version: BoxItemVersion) => {
+            const { modified_by } = version;
+            return {
+                ...version,
+                modified_by: modified_by || PLACEHOLDER_USER,
```
Comment (jeremypress) [inline src/api/Versions.js]: Should we type this with flow?

**Comment by JustinHoldstock** [comment]
Result:
screen shot 2018-08-27 at 4 19 39 pm

File: src/constants.js [hunk 2]
```diff
@@ -236,3 +236,6 @@ export const UNAUTHORIZED_CODE = 403;
 export const VERSION_RESTORE_ACTION = 'restore';
 export const VERSION_DELETE_ACTION = 'delete';
 export const VERSION_UPLOAD_ACTION = 'upload';
+
+/* ------------------ Placeholder Feed Items ------------------------- */
+export const PLACEHOLDER_USER = { type: 'user', id: '2', name: '' };
```
Comment (jeremypress) [inline src/constants.js]: Can we default this without adding this user? There may be cases where it's not actually an anonymous user so this data would be inaccurate.

Comment (JustinHoldstock) [reply src/constants.js]: As discussed, leaving as is.

**Pull request merged**